vocab_list/views.py

```diff
@@ -0,0 +1,4 @@
+from django.shortcuts import render
```

Comment:
Can we delete this and `tests.py` until they are needed?

Reply:
yeah, the views will probably come shortly but may as well remove them both for now.